### PR TITLE
Added the manual camera control code

### PR DESCRIPTION
**Pull request by oziphantom**

### CODE REVIEW

File: src/game/camera.c
Comment (rr-) [inline src/game/camera.c]: Would `CLAMP` macros help here?

File: src/game/camera.c
Comment (rr-) [inline src/game/camera.c:698]: Please avoid double assignments.

File: src/global/types.h
Comment (rr-) [inline src/global/types.h:1760]: Consider renaming these to `angle_override` and `elevation_override`. Please put the comment on a separate line.

File: src/specific/input.c
Comment (rr-) [inline src/specific/input.c]: Let's move this to `LaraControl`.

**Review by rr- — CHANGES_REQUESTED** [review]
Please fix the formatting, fix missing entries in `init.c` and move the input management to `LaraControl`.

File: src/specific/input.c
Comment (rr-) [inline src/specific/input.c:383]: Now we're checking for these twice; once is enough.

**Comment by rr-** [comment]
BTW just to make sure – once we get second approval, I'll merge it by myself; I'll squash the WIP commits into one and provide a consistent commit msg :)

File: src/game/laramisc.c
Comment (thatguyfrombb) [inline src/game/laramisc.c:158]: Does this handle correctly people holding "Camera Up" and "Camera Right" at the same time?

File: src/global/types.h
Comment (thatguyfrombb) [inline src/global/types.h:1760]: As rr- said, different file that needs the change from "additional_angle" to  "angle_override"

**Review by thatguyfrombb — CHANGES_REQUESTED** [review]

File: src/game/camera.c
Comment (oziphantom) [reply src/game/camera.c:38]: however you are not overriding it, it still rotates with Lara. 
So the camera angle  is Lara.y_rot + additional_angle

from line 338: `PHD_ANGLE angle = item->pos.y_rot + Camera.target_angle;`

if we override the angle then it would always face the same way.

Comment (thatguyfrombb) [reply src/game/camera.c:38]: Sorry, I missed that previous message.

**Comment by rr-** [comment]
One last thing: let's add information about this feature to the README.

**Pull request merged**
